api/api-rule-violations-known.list

```diff
@@ -162,6 +162,7 @@ API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,VirtualMachin
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,VirtualMachineInstanceList,Items
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,VirtualMachineInstanceMigrationList,Items
 API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,VirtualMachineInstanceMigrationStatus,Conditions
+API rule violation: list_type_missing,kubevirt.io/client-go/api/v1,VirtualMachineInstanceNetworkInterface,CIDRs
```

Comment:
what about the CIDR?

Reply:
CIDR is not a list type `list_type_missing`.